mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -224,6 +224,8 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_32326 getPreferredEquipmentSlot (Lnet/minecraft/class_1799;)Lnet/minecraft/class_1304;
 		ARG 0 stack
 	METHOD method_33190 canTakeDamage ()Z
+	METHOD method_33579 updateValuesFromSpawnPacket (Lnet/minecraft/class_2610;)V
```

Comment:
```suggestion
	METHOD method_33579 readFromPacket (Lnet/minecraft/class_2610;)V
```
https://github.com/FabricMC/yarn/blob/21w10a/CONVENTIONS.md#serialization
I looked at the code a bit, and yes this method does set up the entity from everything in the packet.